package.json

```
@@ -38,18 +39,28 @@
   },
   "homepage": "https://github.com/mljs/pls",
   "dependencies": {
-    "ml-matrix": "^6.1.2"
+    "ml": "^5.0.0",
```

```suggestion
    "ml": "^5.0.0",
```

Please remove this dependency. We should only depend on single packages like the ones below

---

done

---

I removed dist. Should I also remove lib?

---

> I removed dist.

I think you forgot to push

> Should I also remove lib?

Yes, but I don't see it here.